src/store/modules/auth.js

```diff
@@ -101,7 +101,7 @@ export default {
         );
       });
     },
-    async requestStatus(context, { router, route }) {
+    async requestStatus(context, { route }) {
```

Comment:
If route is the only argument remove the object wrapper and just make it a normal, non destructured argument.